tasty-plutus/src/Test/Tasty/Plutus/Internal/Run.hs

```diff
@@ -14,45 +14,44 @@ import Plutus.V1.Ledger.Api (
  )
 import Plutus.V1.Ledger.Scripts (
   Context,
-  ScriptError,
+  ScriptError (EvaluationError, EvaluationException, MalformedScript),
   runMintingPolicyScript,
   runScript,
  )
 import Safe (lastMay)
 
--- The result of parsing a log from a script emulation
+-- The result of parsing a log from a errored script emulation
 data ScriptResult
```

Comment:
`ScriptResult` now contains only failing alternatives, so it should probably be renamed to `ScriptFailure`.

Reply:
On the second thought, do we really need a new data type at all? What's wrong with just returning the same `ScriptError` we got from Plutus?

Reply:
`ScriptResult` has separate cases for intentional script error and `tasty-plutus` parse error, that's why I didn't use `ScriptError` directly. I'd love to switch that to a better formulation if you have any suggestions.